### ISSUE TITLE
Crash after PR #6

### ISSUE THREAD

**Issue by Andrew-Quebe** [Andrew-Quebe/Planets-Gradle#7]
This happens at app start. Caused by PR #6 

```
06-18 18:32:34.986    3391-3391/com.andrewq.planets E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: com.andrewq.planets, PID: 3391
    java.lang.OutOfMemoryError: Failed to allocate a 3145740 byte allocation with 2021820 free bytes and 1974KB until OOM
            at dalvik.system.VMRuntime.newNonMovableArray(Native Method)
            at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method)
            at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:609)
            at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:444)
            at android.graphics.BitmapFactory.decodeResource(BitmapFactory.java:467)
            at android.graphics.BitmapFactory.decodeResource(BitmapFactory.java:497)
            at com.andrewq.planets.fragments.FragmentPlanets$MyAdapter.getView(FragmentPlanets.java:1196)
            at com.nhaarman.listviewanimations.BaseAdapterDecorator.getView(BaseAdapterDecorator.java:84)
            at com.nhaarman.listviewanimations.swinginadapters.AnimationAdapter.getView(AnimationAdapter.java:133)
            at android.widget.AbsListView.obtainView(AbsListView.java:2344)
            at android.widget.GridView.onMeasure(GridView.java:1060)
            at android.view.View.measure(View.java:17430)
            at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:727)
            at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:463)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.widget.RelativeLayout.measureChild(RelativeLayout.java:697)
            at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:481)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.support.v4.widget.DrawerLayout.onMeasure(DrawerLayout.java:868)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.support.v7.internal.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:124)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1436)
            at android.widget.LinearLayout.measureVertical(LinearLayout.java:722)
            at android.widget.LinearLayout.onMeasure(LinearLayout.java:613)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2560)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2001)
            at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1166)
            at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1372)
            at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1054)
            at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5779)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
            at android.view.Choreographer.doCallbacks(Choreographer.java:580)
            at android.view.Choreographer.doFrame(Choreographer.java:550)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
            at android.os.Handler.handleCallback(Handler.java:739)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5221
```

**Comment by Andrew-Quebe** [comment]
@alorma any ideas?

**Comment by alorma** [comment]
maybe it's just because your i mages are really big.

try to replace the images loading (from BitmapFactyory, to something like Picasso)

**Comment by Andrew-Quebe** [comment]
Since this was your PR, I think it'd be best if you fix it. (I'm not familiar with any of the above mentioned.)

**Comment by alorma** [comment]
Well, first you can rollback, so this will not be in production.

I will try to fix as soon as possible

**Comment by Andrew-Quebe** [comment]
Well, the code isn't live on the production app yet…I'll leave it here. Just 
don't take too long 
![:)](cid:OVgo7hZWEcTBuXF85EYe5WYVduttiDthaQNJ2U5Bxt0ADurIs.smtp.gmail.com)

**Comment by Andrew-Quebe** [comment]
This issue appears to be fixed with PR #8 and/or #9. Will keep open until further notice.

**Comment by Andrew-Quebe** [comment]
All traces of this bug seem to be resolved. @alorma it appears your assistance is no longer required!
